website/docs/r/ssmcontacts_contact.html.markdown

```diff
@@ -45,14 +45,14 @@ resource "aws_ssmcontacts_contact" "example" {
 
 The following arguments are required:
 
-- `alias` - (Required) A unique and identifiable alias for the contact or escalation plan.
+- `alias` - (Required) A unique and identifiable alias for the contact or escalation plan. 1-50 characters, a-z, 0-9, _ (underscore), and - (hyphen).
```

Comment:
Tweaked the phrasing a bit in this suggestion. I also noticed in the [API documentation](https://docs.aws.amazon.com/incident-manager/latest/APIReference/API_SSMContacts_CreateContact.html) that the upper limit appears to be 255 characters.

```suggestion
- `alias` - (Required) A unique and identifiable alias for the contact or escalation plan. Must be between 1 and 255 characters, and may contain alphanumerics, underscores (`_`), and hyphens (`-`).
```